.github/workflows/url-checker.yml

```diff
@@ -1,23 +1,41 @@
 name: URL Checker
 
 on:
-  workflow_dispatch:
+  pull_request:
+    paths:
+      - '5.0/**'
   push:
     branches:
       - master
+    paths:
+      - '5.0/**'
+  workflow_dispatch:
 
 jobs:
   markdown-link-check:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 1
-      - name: link-check
+        uses: actions/checkout@v4
+
+      - name: Set options
+        id: options
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            echo "check-modified-files-only=yes" >> $GITHUB_OUTPUT
+            echo "use-quiet-mode=yes" >> $GITHUB_OUTPUT 

```

Comment:
Would it be clearer to put this outside the if statement if the answer is the same either way or even move it back to being hard coded?

Reply:
Good point, done.